src/apis/sourceimages.js

```diff
@@ -11,27 +11,63 @@ export default (state) => {
   /**
    * Get a list of source images.
    *
+   * By default, listing sourceimages sorts them by created date descending.
+   *
    * ```js
    * rokka.sourceimages.list('myorg')
    * 	 .then(function(result) {})
    * 	 .catch(function(err) {});
    * ```
    *
+   * Searching for images can be achieved using the `search` parameter.
+   * Supported are predefined fields like `height`, `name` etc. but also user metadata.
+   * If you search for user metadata, the field name has to be prefixed with `user:TYPE`.
+   * All fields are combined with an AND. OR/NOT is not possible.
+   *
+   * Example:
+   *
+   * ```js
+   * const search = {
+   *   'user:int:id': '42',
+   *   'height': '64'
+   * }
+   * rokka.sourceimages.list('myorg', { search: search })
+   *   .then(function(result) {})
+   *   .catch(function(err) {});
+   * ```
+   *
+   * The search also supports range and wildcard queries.
+   * Check out [the rokka documentation](https://rokka.io/documentation/references/source-images.html) for more.
+   *
+   * Sorting works with user metadata as well and can be passed as either an array or as a
+   * comma separated string.
+   *
    * @authenticated
-   * @param  {string} organization  name
-   * @param  {number} [limit=null]
-   * @param  {number} [offset=null]
+   * @param  {string}       organization  name
+   * @param  {number}       [limit=null]
```

Comment:
Nah, they're correct. Or at least the least incorrect. 
Tried various options, none I really like:
```
@param {Object} qp Query params
@param {number} [qp.limit=null] Limit
// etc. 
```
Problem with that: there's no qp param

```
@param {{limit: number, offset: number}} qp
```
Same issue: no qp, plus having all the possible things within one line is ugly.